frontend/src/components/Timer/Timer.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
```

Comment:
This is a great start, but we will want to write this out as a "functional" component since that's the way the industry has gone. 

https://reactjs.org/docs/components-and-props.html